webui/bindings.nim

```diff
@@ -249,10 +249,10 @@ proc scriptCleanup*(script: ptr Script) {.cdecl,
     importc: "webui_script_cleanup", webui.}
 proc scriptRuntime*(win: ptr Window; runtime: cuint) {.cdecl,
     importc: "webui_script_runtime", webui.}
-proc getInt*(e: ptr Event): cint {.cdecl, importc: "webui_get_int", webui.}
+proc getInt*(e: ptr Event): int64 {.cdecl, importc: "webui_get_int", webui.}
```

Comment:
Use [`clonglong`](https://nim-lang.org/docs/system.html#clonglong) instead.

Reply:
Use `clonglong` for `returnInt` aswell.

Reply:
True, `clonglong` is better when interacting with a C/C++ library.